### PR TITLE
Bump internal package build number

### PR DESCRIPTION
**Pull request by guominrui**
### Description of the changes:
Bump internal package build number.

### How changes were validated:
Manually tested, and passed the automated tests.

### CODE REVIEW

File: build/pipelines/azure-pipelines.ci-internal.yaml [hunk 1]
```diff
@@ -20,16 +20,24 @@ jobs:
...
   parameters:
     platform: x64
     runsettingsFileName: CalculatorUITests.release.runsettings
 
+- template: ./templates/run-ui-tests.yaml
+  parameters:
+    platform: x86
+    runsettingsFileName: CalculatorUITests.release.runsettings
+    condition: not(eq(variables['Build.Reason'], 'PullRequest'))
+
```
Comment (guominrui) [inline build/pipelines/azure-pipelines.ci-internal.yaml:40]: Add all UI tests into ci-internal, as release pipeline did previously.

File: build/pipelines/azure-pipelines.release.yaml [hunk 2]
```diff
@@ -4,13 +4,6 @@
 # Store and the Windows image. This pipeline relies on Microsoft-internal resources to run.
 #
 
-schedules:
-- cron: "0 7 * * *"
-  displayName: Daily midnight build
-  branches:
-    include:
-    - master
-
```
Comment (guominrui) [inline build/pipelines/azure-pipelines.release.yaml:13]: Stop the schedules, only trigger release pipeline manually after ci-internal passed.

File: build/pipelines/azure-pipelines.release.yaml [hunk 3]
```diff
@@ -26,27 +19,20 @@ jobs:
...
-- template: ./templates/run-ui-tests.yaml
-  parameters:
-    platform: x64
-    runsettingsFileName: CalculatorUITests.release.runsettings
-
-- template: ./templates/run-ui-tests.yaml
-  parameters:
-    platform: x86
-    runsettingsFileName: CalculatorUITests.release.runsettings
-
```
Comment (guominrui) [inline build/pipelines/azure-pipelines.release.yaml:49]: All the UI tests will be covered in ci-internal.

File: build/pipelines/azure-pipelines.ci-internal.yaml [hunk 4]
```diff
@@ -20,16 +20,24 @@ jobs:
 - template: ./templates/build-app-internal.yaml
   parameters:
     platform: x86
+    condition: not(eq(variables['Build.Reason'], 'PullRequest'))
```
Comment (hanzhang54) [inline build/pipelines/azure-pipelines.ci-internal.yaml:23]: We should also run the ci internal pipeline for pull request reason. The condition in build-app-internal.yaml will use the non-release version for build and pass the UI test as well.

File: build/pipelines/azure-pipelines.release.yaml (hunk 2 above)
Comment (hanzhang54) [reply build/pipelines/azure-pipelines.release.yaml:13]: It seems to be not necessary to remove this schedule. This schedule is for the master branch, and we should check the ci result before merge any PR into master.

Comment (guominrui) [reply build/pipelines/azure-pipelines.release.yaml:13]: If having the schedule here, there is chance that maybe release pipeline starts before ci-internal done.

File: build/pipelines/azure-pipelines.ci-internal.yaml (hunk 4 above)
Comment (guominrui) [reply build/pipelines/azure-pipelines.ci-internal.yaml]: I have uploaded a new commit to resolve this.

**Review by hanzhang54 — APPROVED** [review]

**Pull request merged**